### PR TITLE
[pfs] Update run api in prompt flow service

### PR DESCRIPTION
**Pull request by lalala123123**
# Description

### Submit run
image

### Get Run
image

### Update Run
image

### List Run
image

### Child runs
image

### Node runs
image

### Visualize
image

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: src/promptflow/promptflow/_sdk/_service/apis/run.py [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
-from promptflow._sdk._constants import FlowRunProperties, get_list_view_type
+from promptflow._sdk._constants import get_list_view_type
 from promptflow._sdk._errors import RunNotFoundError
 from promptflow._sdk.operations._local_storage_operations import LocalStorageOperations
 from promptflow._sdk.operations._run_operations import RunOperations
-from promptflow.contracts._run_management import RunMetadata
 
 api = Namespace("Runs", description="Runs Management")
 
+# Define get connection request parsing
```
Comment (wangchao1230) [inline src/promptflow/promptflow/_sdk/_service/apis/run.py:16]: connection -> run

File: src/promptflow/promptflow/_sdk/_service/apis/run.py [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
+update_run_parser = api.parser()
+update_run_parser.add_argument("display_name", type=str, location="args", required=False)
+update_run_parser.add_argument("description", type=str, location="args", required=False)
+update_run_parser.add_argument("tags", type=str, location="args", required=False)
+update_run_parser.add_argument("display_name", type=str, location="args", required=False)
+
+# Response model of connection operation
```
Comment (wangchao1230) [inline src/promptflow/promptflow/_sdk/_service/apis/run.py:23]: connection -> run

File: src/promptflow/promptflow/_sdk/_service/apis/run.py (hunk 1 above)
Comment (lalala123123) [reply src/promptflow/promptflow/_sdk/_service/apis/run.py:16]: fix it

File: src/promptflow/promptflow/_sdk/_service/apis/run.py (hunk 2 above)
Comment (lalala123123) [reply src/promptflow/promptflow/_sdk/_service/apis/run.py:23]: fix it

File: src/promptflow/promptflow/_sdk/_service/apis/run.py [hunk 3]
```diff
@@ -42,17 +61,80 @@ def get(self):
...
+        run_name = run_dict.get("name", None)
+        if not run_name:
+            run = RunEntity(**run_dict)
+            run_name = run._generate_run_name()
+            run_dict["name"] = run_name
+        with tempfile.TemporaryDirectory() as temp_dir:
+            run_file = Path(temp_dir) / "batch_run.json"
+            with open(run_file, "w") as f:
+                json.dump(run_dict, f)
+            cmd = f"pf run create --file {run_file}"
```
Comment (wangchao1230) [inline src/promptflow/promptflow/_sdk/_service/apis/run.py:79]: comment why we need to do this

Comment (lalala123123) [reply src/promptflow/promptflow/_sdk/_service/apis/run.py:79]: fix in next pr

**Review by wangchao1230 — APPROVED** [review]

**Review by zhengfeiwang — APPROVED** [review]

**Pull request merged**